### DISCUSSION TITLE
Dropdown menu on right-click?

### DISCUSSION THREAD

**Discussion by mark1502** [saadeghi/daisyui#3031] [Q&A] [answered]
I was looking at the dropdown menu and the examples are all activated on a button click?  Is it possible to activate a dropdown menu on a right-click but not a left-click?  Or can it be activated similar to a modal?

**Comment by saadeghi** [comment, marked as answer]
It depends on your framework. Here's a simple JS code:  
https://codepen.io/saadeghi/pen/yLWyByy
